main.css

```diff
@@ -0,0 +1,53 @@
+* {
+    padding: 0;
+    margin: 0;
+    box-sizing: border-box;
+  }
+  
+  body {
```

Comment:
Please delete this body if you are not using it.

Reply:
- [OPTIONAL] Nice one @AbdifatahYasin1, but try as much as you can to include `WHY` the student is asked to make use of a suggestion as this would make it clearer for the student to know why he or she is being asked to do a certain thing. And  if possible, you can also direct the student to a resource where more information could be got for  more clarification.